src/modules/commander/commander_params.c

```diff
@@ -799,9 +799,21 @@ PARAM_DEFINE_INT32(NAV_RCL_ACT, 2);
 
 /**
  * Flag to enable obstacle avoidance
- * Temporary Parameter to enable interface testing
  *
  * @boolean
  * @group Mission
  */
-PARAM_DEFINE_INT32(COM_OBS_AVOID, 0);
\ No newline at end of file
+PARAM_DEFINE_INT32(COM_OBS_AVOID, 0);
+
+/**
+ * Set onboard controller bootup timeout
```

Comment:
I would call it `onboard computer` or `companion computer`.

Reply:
Strongly agree. FWIW IMO terminology for onboard/offboard is unhelpful because it is all from perspective of FC, but people think in terms of vehicles.

Is this a general companion computer timeout or an avoidance system running on companion computer timeout? 

Assuming the later I'd do something like:

>Set avoidance system bootup timeout.
>
> The avoidance system running on the companion computer is expected to boot within this time and start providing trajectory points. If no avoidance system is detected a MAVLink warning message is sent.

Note, I have tried to explain how it detects the companion computer.

Reply:
Sorry, I've lost that comment line during my changes.
@hamishwillee thanks for your suggestion, makes sense.

To answer your question: Yes, it is exactly the time within the avoidance system is expected to be running (formally defined as` companion computer boot time + avoidance system startup time`).

To this aim i thing that makes sense to change the name parameter to a more consistent too, i.e `COM_OA_BOOT_T` or `COM_OBS_AVOID_T`.

I think that the second one would be more consistent with `COM_OBS_AVOID`.
What do you think? @hamishwillee @julianoes

Reply:
I would probably vote `COM_OA_BOOT_T` because it's more self explanatory.